addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -1378,12 +1378,13 @@ export class PosStore extends Reactive {
         const baseUrl = this.session._base_url;
         return order.export_for_printing(baseUrl, headerData);
     }
-    async printReceipt(order = this.get_order()) {
+    async printReceipt(basic = false, order = this.get_order()) {
```

Comment:
```suggestion
    async printReceipt( {basic = false, order = this.get_order() } = {}) {
```
this makes it more clear when you call the option. Doing `printReceipt(true)` does not suggest what the `true` means. Doing `printReceipt({ basic: true })`  is more clear